### PR TITLE
Quick Add UI

### PR DESCRIPTION
**Pull request by thehabes**
Moved the little + button to the top of the dropdown/multiselect and made sure it touched what it applies to.

The modal it triggers is now a popup, kind of like prompt().  

Deployed to dev, test at http://lived-religion-dev.rerum.io/deer-lr/experience.html

### CODE REVIEW

**Comment by cubap** [comment]
delayed while devstore.rerum.io is down.

**Comment by thehabes** [comment]
Ahh whoops.  I will get the tinydev.rerum.io and awstest.rerum.io squared away.

File: web/js/deerInitializer.js [hunk 1]
```diff
@@ -54,19 +54,20 @@ DEER.TEMPLATES.Sense = function(obj, options = {}) {
...
+        let quickAddTmpl = `<a title="Click here to add a new entity by name to this collection." class="quick tag bg-primary text-white is-small pull-right" onclick="LR.ui.toggleEntityAddition(event, this.nextElementSibling)">&#x2b;</a>
+        <div class="card quickAddEntity bg-light is-hidden">
+            <label>Supply a name or label for this entity</label>
+            <a class="closeQuickAdd quick tag bg-primary text-white is-small pull-right" onclick="LR.ui.toggleEntityAddition(event, this.parentElement)"> &#8722; </a>
+            <input class="" type="text" />
+            <a class="tag bg-primary text-white" onclick="LR.utils.quicklyAddToCollection(event, '${whichCollection}', null, 'Place')">Add</a>
         </div>`
-        let tmpl = `<select class="locDropdown" oninput="this.parentElement.previousElementSibling.value=this.options[this.selectedIndex].value">`
+        let tmpl = `${quickAddTmpl}<select class="locDropdown" oninput="this.parentElement.previousElementSibling.value=this.options[this.selectedIndex].value">`
```
Comment (cubap) [inline web/js/deerInitializer.js:64]: I prefer this being in the <label>, but that can be for the future.

File: web/js/deerInitializer.js [hunk 2]
```diff
@@ -110,21 +112,22 @@ DEER.TEMPLATES.objectsAsDropdown = function(obj, options = {}) {
...
-        <div title="Quickly create a new entity for this collection by supplying a name or label." class="row quickAddEntity bg-light is-hidden">
-            <span class="">Label:</span>
-            <input class="bg-grey text-white" type="text" />
-            <a class="tag bg-primary text-white is-small" onclick="LR.utils.quicklyAddToCollection(event, '${whichCollection}', this.closest('deer-view').querySelector('.selectedEntities'), 'Place')">Add</a>
+        let quickAddTmpl = `<a title="Click here to add a new entity by name to this collection." class="quick tag bg-primary text-white is-small pull-right" onclick="LR.ui.toggleEntityAddition(event, this.nextElementSibling)">&#x2b;</a>
+        <div class="card quickAddEntity bg-light is-hidden">
+            <label>Supply a name or label for this entity</label>
+            <a class="closeQuickAdd quick tag bg-primary text-white is-small pull-right" onclick="LR.ui.toggleEntityAddition(event, this.parentElement)"> &#8722; </a>
```
Comment (cubap) [inline web/js/deerInitializer.js:118]: The "-" is a little less obvious. Probably an "x" or "&times;" is best.

**Pull request merged**